src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -1119,16 +1130,22 @@ void Replayer<I>::handle_copy_image(int r) {
     m_last_snapshot_sync_seconds = duration.sec();
 
     if (g_snapshot_perf_counters) {
-      g_snapshot_perf_counters->inc(l_rbd_mirror_snapshot_replay_bytes,
+      g_snapshot_perf_counters->inc(l_rbd_mirror_snapshot_sync_bytes,
                                     m_snapshot_bytes);
-      g_snapshot_perf_counters->inc(l_rbd_mirror_snapshot_replay_snapshots);
+      g_snapshot_perf_counters->inc(l_rbd_mirror_snapshot_snapshots);
       g_snapshot_perf_counters->tinc(
-        l_rbd_mirror_snapshot_replay_snapshots_time, duration);
+        l_rbd_mirror_snapshot_sync_time, duration);
     }
     if (m_perf_counters) {
-      m_perf_counters->inc(l_rbd_mirror_snapshot_replay_bytes, m_snapshot_bytes);
-      m_perf_counters->inc(l_rbd_mirror_snapshot_replay_snapshots);
-      m_perf_counters->tinc(l_rbd_mirror_snapshot_replay_snapshots_time, duration);
+      m_perf_counters->inc(l_rbd_mirror_snapshot_sync_bytes,
+                           m_snapshot_bytes);
+      m_perf_counters->inc(l_rbd_mirror_snapshot_snapshots);
+      m_perf_counters->tinc(l_rbd_mirror_snapshot_sync_time,
+                            duration);
+      m_perf_counters->tset(l_rbd_mirror_snapshot_last_sync_time,
+                           duration);
+      m_perf_counters->set(l_rbd_mirror_snapshot_last_sync_bytes,
+                           m_last_snapshot_bytes);
```

Comment:
For consistency with `l_rbd_mirror_snapshot_sync_bytes`:
```suggestion
                           m_snapshot_bytes);
```